huaweicloud/services/cfw/resource_huaweicloud_cfw_dns_resolution.go

```diff
@@ -0,0 +1,395 @@
+package cfw
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API CFW GET /v1/{project_id}/dns/servers
+// @API CFW PUT /v1/{project_id}/dns/servers
+func ResourceDNSResolution() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDNSResolutionCreate,
+		ReadContext:   resourceDNSResolutionRead,
+		UpdateContext: resourceDNSResolutionUpdate,
```

Comment:
the update method is same as the create method, just keep one

Reply:
fixed